src/binary.js

```diff
@@ -0,0 +1,5 @@
+const toBinary = number => {
+
+    if(number > 0) return number.toString(2)
+    return (number >>> 0).toString(2)
+}
```

Comment:
Hi @Kevinemug 
Good Job on completing these exercises
this code looks good just remove the space at the beginning of the function and you're good to go